core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -803,7 +803,7 @@ class ZkUtils(val zkClient: ZkClient,
         createParentPath(BrokerSequenceIdPath, acls)
         try {
           zkClient.createPersistent(BrokerSequenceIdPath, "", acls)
-          0
+          1
```

Comment:
Thanks for the review. Yes, This call shouldn't  return a number twice  and it should start from 1.

As i mentioned in the JIRA,  we are already creating the zknode during server startup, and catch block should not get executed.  I will remove unnecessarily code and upload new patch.

I will load